sys/net/gnrc/netif/cc1xxx/gnrc_netif_cc1xxx.c

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
Mhh, following https://github.com/RIOT-OS/RIOT/pull/9917#issuecomment-450176314 I'm now not sure `sys/net/gnrc/netif/cc1xxx` is the right place for this. Maybe `drivers/cc1xxx_common`? This would also allow for some "cross polination" of code for the two `cc1xxx` drivers, if some code duplication appears.

Reply:
> Maybe `drivers/cc1xxx_common`?

Honestly, I do not have the deep understanding about RIOT's structure to have a profound opinion for or against moving it. But `drivers/cc1xxx_common` sounds good to me, so I will move it there.

> This would also allow for some "cross polination" of code for the two cc1xxx drivers, if some code duplication appears.

This would be nice indeed and is good argument for `drivers/cc1xxx_common` :-)

Reply:
Done. I will rebase https://github.com/maribu/RIOT/commits/cc110x_rewrite2 on top of this to test it